sympy/solvers/ode/single.py

```diff
@@ -794,26 +794,28 @@ def _get_general_solution(self, *, simplify: bool = True):
 class SecondAutonomousNth(SinglePatternODESolver):
     r"""
     The general second order autonomous differential equation of the form:
-    .. math :: x''(t) + f(x)*x'(t) + g(x) = 0
 
-    This solver is added for the special case in this genre:
+    .. math :: x''(t) = g(x'(t), x(t))
+
+    This solver is added for the special case in the above genre:
+
     .. math :: f''(x) = g(f(x)) f'(x)^n
 
-    This special case can be solved using by writing the second derivative in
-    form involving a power of x' and on rewriting the second derivative, rearranging.
+    Explanation
+    ===========
 
-    For proof of the results, kindly refer the below link:
-    https://en.wikipedia.org/wiki/Autonomous_system_(mathematics)
-    http://eqworld.ipmnet.ru/en/solutions/ode/ode0301.pdf
+    This special case can be solved using by writing the second derivative in
+    form involving a power of f'(x) and on rewriting the second derivative, then rearranging.
+    This solver works for any arbitrary value of n defined above.
 
-    Caution: If n is even except for 2, kindly set simplify = False in dsolve()
+    Extras: For faster results, kindly set simplify = False in dsolve().
```

Comment:
Now, we don't need to use `simplify` for any case as such. Though we can use it to get results faster as some solutions come in form of integrals that can't be solved further, and we can avoid `sympy` to not try simplifying it further, as it consumes a lot of time to find that it can't be simplified further by `sympy` like in `f(x).diff(x, 2) - exp(f(x)) + log(f(x))`.